tests/yaml/capitalization.yaml

```diff
@@ -9,7 +9,7 @@ table: |
   uplow Yy 5
   uplow Zz 6
   sign - 23
-  seqdelimiter -
+  seqdelimiter - # Has no effect with capitals

```

Comment:
What do you mean with this comment? That the could be removed?

Reply:
Yes, I removed it, and it had no effect. Perhaps, it is not because of seqdelimiter having no effect, but rather whatever is wrong with capsphrase.

Reply:
I don't think there is anything wrong with `capsphrase`. Besides it is not even used in this test.

Let's just remove this line from the test.

I've looked a bit at the code and it seems `seqdelimiter` is not used for applying emphasis, except in one place but I suspect that is a mistake.